docs/docsite/rst/conf.py

```diff
@@ -65,6 +70,7 @@
     'notfound.extension',
     'sphinx_antsibull_ext',  # provides CSS for the plugin/module docs generated by antsibull
     'sphinx_copybutton',
+    'sphinx_reredirects', # redirects pages that have been restructured or removed
```

Comment:
I meant that loading this extension could be made conditional by placing it into the `setup()` hook, similar to the standalone extension.